### ISSUE TITLE
Unable to init sdk for side-loaded UWP app

### ISSUE THREAD

**Issue by jimmyleeking** [dji-sdk/Windows-SDK#33]
my app is working fine on my development Windows 10  PC,but when i release my app,that always crash when i init sdk ,that it's so bad for my customer ,please checkout issue.

CALL STACK:

[2: System.Runtime.InteropServices.ExternalInterop.RoGetActivationFactory(String, Guid&, IntPtr&) + 0x197	0](https://app.bugsplat.com/groupstacks/?stackId=3&frameCount=1)

- [2: System.Runtime.InteropServices.FactoryCache.GetActivationFactoryInternal(String, RuntimeTypeHandle, ContextEntry) + 0x4a	0](https://app.bugsplat.com/groupstacks/?stackId=3&frameCount=2)
  - [2: System.Runtime.InteropServices.FactoryCache.GetActivationFactory(String, RuntimeTypeHandle, Boolean) + 0x6c	0](https://app.bugsplat.com/groupstacks/?stackId=3&frameCount=3)
    - [2: System.Runtime.InteropServices.McgMarshal.GetActivationFactory(String, RuntimeTypeHandle) + 0x24	0](https://app.bugsplat.com/groupstacks/?stackId=3&frameCount=4)
      - [2: System.Runtime.InteropServices.McgMarshal.ActivateInstance(String) + 0x36	0](https://app.bugsplat.com/groupstacks/?stackId=3&frameCount=5)
        - [2: DJIWindowsWrapper.DJIWindowsNetworkHandle..ctor() + 0x1c	0](https://app.bugsplat.com/groupstacks/?stackId=3&frameCount=6)
          - [2: DJI.WindowsSDK.DJISDKManager..ctor() + 0x27	0](https://app.bugsplat.com/groupstacks/?stackId=3&frameCount=7)
            - [2: DJI.WindowsSDK.DJISDKManager..cctor() + 0x55d1	0](https://app.bugsplat.com/groupstacks/?stackId=3&frameCount=8)
              - [2: System.Runtime.CompilerServices.ClassConstructorRunner.EnsureClassConstructorRun(StaticClassConstructionContext*) + 0xf1	0](https://app.bugsplat.com/groupstacks/?stackId=3&frameCount=9)
                - [2: --- End of inner exception stack trace ---	0](https://app.bugsplat.com/groupstacks/?stackId=3&frameCount=10)
                  - [2: System.Runtime.CompilerServices.ClassConstructorRunner.EnsureClassConstructorRun(StaticClassConstructionContext*) + 0x1ad	0](https://app.bugsplat.com/groupstacks/?stackId=3&frameCount=11)
                    - [2: System.Runtime.CompilerServices.ClassConstructorRunner.CheckStaticClassConstruction(Void*, StaticClassConstructionContext*) + 0xf	0](https://app.bugsplat.com/groupstacks/?stackId=3&frameCount=12)
                      - [2: DJI.WindowsSDK.DJISDKManager.get_Instance() + 0x10	0](https://app.bugsplat.com/groupstacks/?stackId=3&frameCount=13)
                        - [2: FeimaSmartAirport4UWP.FeimaDroneProxy.d.MoveNext() + 0x21	0](https://app.bugsplat.com/groupstacks/?stackId=3&frameCount=14)

**Comment by hissain** [comment]
@jimmyleeking were you able to fix the issue?

**Comment by jimmyleeking** [comment]
> 
> 
> @jimmyleeking were you able to fix the issue?

Nope, I think it's should be an bug for DJI SDK (but developer may be not agree it,I already sent email to request answer )

I already quit DJI SDK for Windows ,that 's bug SDK ,you should not use it for real production ,that would be crazy,unstable.

and developer not willing to reply your  any issue.
